repo/packages/C/cassandra/200/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "packagingVersion": "4.0",
+  "upgradesFrom": [
+    "2.1.0-3.0.16"
+  ],
+  "downgradesTo": [
+    "2.1.0-3.0.16"
+  ],
+  "minDcosReleaseVersion": "1.10",
```

Comment:
Is this correct? @triclambert @benclarkwood We have at another universe PR minDcosReleaseVersion 1.10.

Reply:
minDcosReleaseVersion should be 1.9